controls/srg_ctr/SRG-APP-000219-CTR-000550.yml

```diff
@@ -5,4 +5,6 @@ controls:
   title: {{{ full_name }}} must protect authenticity of communications sessions
     with the use of FIPS-validated 140-2 or 140-3 security requirements for cryptographic
     modules.
-  status: pending
+  rules:
+  - fips_mode_enabled_on_all_nodes
+  status: automated
```

Comment:
This is automated in the sense that if the cluster comes up and assuming the user has set the appropriate keys, the nodes will boot in FIPS mode.

Not that we've automated a check for it, correct?

Reply:
we do have an automated check actually, it amounts to:
```
 oc get machineconfig -o json | jq '[.items[] | select(.metadata.name | test("^rendered-worker-[0-9a-z]+$|^rendered-master-[0-9a-z]+$"))] | map(.spec.fips == true)'

```